Orbitersdk/samples/ProjectApollo/src_sys/apolloguidance.cpp

```diff
@@ -542,6 +561,12 @@ void ApolloGuidance::LoadState(FILEHANDLE scn)
 			vagc.Trap31B = state.u.Trap31B;
 			vagc.Trap32 = state.u.Trap32;
 		}
+		else if (!strnicmp(line, "ADDSTATE", 8)) {
```

Comment:
Why `ADDSTATE` exactly?

Reply:
Took me a second too. My guess is it's short for "Additional State".

Reply:
...and something like "STATE2" can't easily work because of how the `scanf()` reading is set up. `STATE2` would be matched by the preceding `else if (!strnicmp (line, "STATE", 5)) {` check.

Reply:
@ThymoNL  Haha yeah, Mike got it right. It's ADDSTATE for additional AGC state. STATE2 can work if functions are used that check on the whole string, which we do in other save/load functions but not here yet. So I couldn't be bothered rewriting the loading function enough so that I could use STATE and STATE2. If you have a better idea for a name, now we can still change it. Once this gets merged we shouldn't do that anymore for backwards compatibility.

Reply:
I get it. Using this name does limit us to only this extra state though. I expect this will be enough for quite a while, but surely that was also the case when AGCSTATE was made initially. How would we add a new state when this one runs out? You'd have to get creative with naming that one, or allow numbers to be used.

Reply:
Alright, I will change it to STATE and STATE2. Just have to rework the loading of those a bit, hopefully without breaking everything!

Reply:
In my testing it looks good, but I am always a bit hesitant with int to unsigned int conversions, which can't be avoided here.

else if (papiReadScenario_int(line, "STATE", inttemp))

inttemp is an integer, because that's what papiReadScenario_int reads.

state.word = inttemp;

word is unsigned long.

I check with putting -1 as STATE in the scenario and it reads (as I believe) correctly as all bits of STATE being set to 1. Any objection to this solution?